addons/payment_mollie_official/models/payment_acquirer.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+from mollie.api.client import Client
+from odoo import models, fields, api, _
+from odoo.http import request
+
+import base64
+import requests
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    _mollie_client = Client()
+
+    provider = fields.Selection(selection_add=[('mollie', 'Mollie')])
+    mollie_api_key_test = fields.Char(
+        'Mollie Test API key', size=40, required_if_provider='mollie',
+        groups='base.group_user')
+    mollie_api_key_prod = fields.Char('Mollie Live API key', size=40,
+                                      required_if_provider='mollie',
+                                      groups='base.group_user')
+    dashboard_url = fields.Char(string="Dashboard URL")
```

Comment:
I don't want any kind of links to the customer's dashboard if this is not strictly necessary. I assume this is something you added for user UX but this is not consistent with other providers.

Reply:
The `dashboard_url` field has been dropped.